docs/reference/search/search-your-data/paginate-search-results.asciidoc

```diff
@@ -28,29 +21,180 @@ GET /_search
 }
 ----
 
-By default, you cannot page through more than 10,000 documents using the `from`
-and `size` parameters. This limit is set using the
-<<index-max-result-window,`index.max_result_window`>> index setting.
+Avoid using `from` and `size` to page too deeply or request too many results at
+once. Search requests usually span multiple shards. Each shard must load its
+requested hits and the hits for any previous pages into memory. For deep pages
+or large sets of results, these operations can significantly increase memory and
+CPU usage, resulting in degraded performance or node failures.
+
+By default, you cannot use `from` and `size` to page through more than 10,000
+hits. This limit is a safeguard set by the
+<<index-max-result-window,`index.max_result_window`>> index setting. If you need
+to page through more than 10,000 hits, use the <<search-after,`search_after`>>
+parameter instead.
+
+WARNING: {es} uses Lucene's internal doc IDs as tie-breakers. These internal doc
+IDs can be completely different across replicas of the same data. When paging
+search hits, you might occasionally see that documents with the same sort values
+are not ordered consistently.
+
+[discrete]
+[[search-after]]
+=== Search after
+
+You can use the `search_after` parameter to retrieve the next page of hits
+using a set of <<sort-search-results,sort values>> from the previous page.
+
+Using `search_after` requires multiple search requests with the same `query` and
+`sort` values. If a <<near-real-time,refresh>> occurs between these requests,
+the order of your results may change, causing inconsistent results across pages. To
+prevent this, you can create a <<point-in-time-api,point in time (PIT)>> to
+preserve the current index state over your searches.
+
+IMPORTANT: We don't recommend creating PITs for each user request. Each PIT
+requires additional disk space and file handles. If you want to paginate search
+results for user requests, use `search_after` without a PIT.
```

Comment:
That's not entirely true. I know we had such recommendations for `search_after` but I think we should remove it now. The goal of the PIT is to provide a way to share the readers internally so that's misleading imo.

Reply:
Good to know. Removed with df7ca438f5ecda1ab49f40a84ec2716f5280edcf. Thanks for the correction!